### DISCUSSION TITLE
Machine readable output in RSS format

### DISCUSSION THREAD

**Discussion by stevepereira** [osmoscraft/osmosfeed#12] [Ideas] [answered]
Love this, it would be great to generate a consolidated RSS feed that could be used elsewhere (like my newsletter app, etc, but maybe there’s a better tool to do that)

**Comment by chuanqisun** [comment]
The app focuses on the final reading experience so I didn't support RSS as an output format. I wonder if this is what you are looking for: http://www.rssmix.com/ ?

**Comment by chuanqisun** [comment]
This issue is originally brought up in #1.

**Comment by chuanqisun** [comment, marked as answer]
Hey @stevepereira, with #25, the tool is now creating Atom feed as part of the build process. [Documentation](https://github.com/osmoscraft/osmosfeed/blob/master/docs/headless-usage-guide.md).

Let me know if this addresses your case. Thank you again!
